services/ui/app/ui_wattlab_studio.py

```
@@ -0,0 +1,126 @@
+        v = (os.environ.get(key) or "").strip()
+        if v:
+            return v
+    return None
+
+
+def _try_import_page(mod_name: str):
+    try:
+        import importlib
+
+        return importlib.import_module(f"wattlab.studio.pages.{mod_name}")
+    except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)